config/plot_porkchop.json.empty

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2014-2016 Kartik Kumar, Dinamica Srl (me@kartikkumar.com)
+// Copyright (c) 2016 Enne Hekma, Delft University of Technology (ennehekma@gmail.com)
+// Distributed under the MIT License.
+// See accompanying file LICENSE.md or copy at http://opensource.org/licenses/MIT
+{
+    // Path to SQLite database containing scan data.
+    "database"                  : "",
+
+    // Directory where scan figure is stored.
+    "output_directory"          : "",
+
+    // Departure and arrival objects. 
+    // If empty the departure and arrival objects of the lowest dV in the database is chosen. 
```

Comment:
can you please change dv to deltaV, its easier to understand that way. Also not just here, but everywhere in this file. Thanks.